server/controllers/finance/feeCenter.js

```diff
@@ -0,0 +1,263 @@
+/**
+* The /fee_centers HTTP API endpoint
+*
+* @module finance/feeCenters
+*
+* @description This controller is responsible for implementing all crud and others custom request
+* on the fee_center table through the `/fee_centers` endpoint.
+*
+* @requires lib/db
+**/
+
+
+var db = require('../../lib/db');
+var NotFound = require('../../lib/errors/NotFound');
+
+/**
+* Returns an array of fee centers
+*
+* @param {object} req The express request object
```

Comment:
I think at this point, all developers can be expected to know how `expressjs` works.  There is no need to document another library's parameters.